scripts/run-proxies.sh

```diff
@@ -62,6 +62,30 @@ print_usage() {
   echo "  -n, --dry             Dry run"
 }
 
+show_help() {
+  cat <<EOF
+Usage:
+  ./scripts/run-proxies.sh [OPTIONS] PROXY1 PROXY2 ...
+
+Options:
+  -f, --remove-containers, --force
+    Remove containers on start and on fail
+  -s, --remove-containers-on-start
+    Remove containers on start
+  -x, --remove-containers-on-fail
+    Remove containers on fail
+  -i, --interval
```

Comment:
I used this when developing to prevent mega terminal spam while in a restart-loop. I want there to be some interval between restart to prevent this. Since the interval should be there, why not expose it in the CLI interface?

I'm neither for, nor against it in the CLI interface.